Tests/CarthageKitTests/ResolverSpec.swift

```diff
@@ -259,76 +259,78 @@ class ResolverSpec: QuickSpec {
 					]
 				}
 
-				// Only the new resolver passes the following tests. Will change to non-pending when checked in
-				pending("should resolve a subset when given specific dependencies that have constraints") {
-					let db: DB = [
-						github1: [
-							.v1_0_0: [
-								github2: .compatibleWith(.v1_0_0),
+				// Only the new resolver passes the following tests.
+				if resolverType == NewResolver.self {
```

Comment:
Recommend ignoring whitespace for viewing the diffs in this file, as it's mostly a large indentation